app/helpers/wowza_helper.rb

```diff
@@ -0,0 +1,44 @@
+module WowzaHelper
+  #---
+  # Builds Wowza URL
+  #
+  # @param obj_id
+  # @param file_set_id
+  # @param fedora_file
+  # @media_type: 'audio', 'video'
+  # @return string or nil
+  #---
+
+  def grabWowzaURL(obj_id, file_set_id, fedora_file, media_type, request_ip)
```

Comment:
I know this is pulling over existing code, but we should probably clean it up a bit since we have the chance.

This method name should be changed to use the ruby method convention. And grab isn't particularly valuable. It could simply be `wowza_url` or something like `encrypted_streaming_url`, or something like that.